packages/manager/modules/vrack/src/dashboard/vrack.constant.js

```diff
@@ -66,6 +67,7 @@ export const TYPE_SERVICE = {
   ipv6: 'ipv6',
 };
 
+// ALL ELIGIBLE SERVICES FOR VRACK
```

Comment:
Commit message says remove vrackServices from eligible list and you are adding vrackServices to eligible SERVICES list. Is the commit message or comment is wrong ?